CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 0.11.0 (03-29-2021)
```

Comment:
you can consider adding hyperlink for each version and align with https://keepachangelog.com/en/1.1.0/

Reply:
Following keepachangelog.com sounds great! Will do that in a different PR. 

What would you link the version to? Github release? PYPI package?

Reply:
Whatever you find more suitable. I have no strong preferences TBH 😉 
https://keepachangelog.com/en/1.1.0/ proposes the diff between releases (look at the bottom of the example changelog)